src/main/perf/LineFileDocs.java

```diff
@@ -357,7 +357,11 @@ public static int idToInt(String id) {
     DocState(boolean storeBody, boolean tvsBody, boolean bodyPostingsOffsets, boolean addDVFields, int vectorDimension) {
       doc = new Document();
 
-      title = new StringField("title", "", Field.Store.NO);
+      FieldType titleFieldType = new FieldType(TextField.TYPE_NOT_STORED);
+      // nocommit should this option be made configurable?
+      titleFieldType.setOmitNorms(false);
+      title = new Field("title", "", titleFieldType);
```

Comment:
We should use `titleTokenized` to benchmark CombinedFieldsQuery rather than `title`?

Reply:
Ah yes! Updated.